### ISSUE TITLE
Reports subpage doesn't work with PostgreSQL

### ISSUE THREAD

**Issue by BenParizek** [barrelstrength/craft-sprout-reports#22]
Accessing the "Reports" sub-page in the CP fails with an exception when using PostgreSQL as the database, as Pg folds all unquoted identifiers to lowercase. Any table or column with upper or mixed case names must always be double-quoted in order to be properly queried from PostgreSQL.

More info: [15931431643](https://app.intercom.io/a/apps/dfa024eb83c11784227312bba407fb3d35e00fa7/inbox/inbox/714050/conversations/15931431643)

### Notify

- [x] [15931431643](https://app.intercom.io/a/apps/dfa024eb83c11784227312bba407fb3d35e00fa7/inbox/inbox/714050/conversations/15931431643)

**Comment by pdaleramirez** [comment]
Fixed the postgres sql bug by adding brackets to the column name https://github.com/barrelstrength/craft-sprout-base/commit/3ac1e3f8c80af0dabf3fbd59e1db883c6177c542

**Comment by tonyclemmey** [comment]
Hi @BenParizek ,

I am experiencing a similar issue if not the same in regards to report subpages not showing. If I create a new Twig template, and access it directly with its link 'admin/sprout-reports/reports/2/edit/6213' I can view it but subpages under reports do not exist/ show.

```
CraftCMS 3.6.17
Sprout Reports 1.5.10
pg_config --version PostgreSQL 13.3 (Ubuntu 13.3-1.pgdg20.04+1)
psql --version psql (PostgreSQL) 13.3 (Ubuntu 13.3-1.pgdg20.04+1)
```

Extra steps taken, uninstall the plugin, reinstall the plugin, create a new report, the same issue.

**Comment by BenParizek** [comment]
@tonyclemmey Looks like this was an issue with my understanding of how Craft's :empty: shorthand is handled by postgres. This is now fixed in sprout-base-reports v1.5.11

I'll get that out in a coming release but for now can you `composer remove` Sprout Reports and re-run the composer installation for the Sprout Reports package to make sure it pulls in the latest barrelstrength/sprout-base-reports v1.5.11.

If you have any existing reports that are in the database and not showing, you will need to update their `sproutreports_reports.emailColumn` value to be NULL instead of EMPTY. And I apologize for the manual update process on this one. Here's an SQL statement to update the db if helpful:

UPDATE "sproutreports_reports"
SET "emailColumn" = NULL;
WHERE "emailColumn" = '';

Let me know if you have any questions.

**Comment by tonyclemmey** [comment]
@BenParizek Thanks.

I also had to delete dashboard `widgets` as well as the `sproutreports_reports.emailColum` EMPTY => NULL change.

`DELETE FROM "widgets" WHERE type LIKE '%[barrelstrength\sproutreports\widgets]%';`

Seems to have done the trick. 

Cheers
